app/search/gleaner.py

```diff
@@ -77,7 +77,7 @@ def build_query(text_query="", filter_query="", page_number=1):
                     ?geo a schema:GeoCoordinates .
                     ?geo schema:longitude ?lon.
                     ?geo schema:latitude ?lat .
-                    BIND(concat('str(?lat), " ", str(?lon)') as ?spatial_coverage_point )  .
+                    BIND(concat(str(?lat), " ", str(?lon)) as ?spatial_coverage_point )  .
```

Comment:
This is a bug! It was causing that 'numbers must be real' thing I saw yesterday, because the string wasn't properly escaped. Story of our lives this week I guess.